src/Translator.php

```diff
@@ -63,7 +64,7 @@ public function exists($key)
      */
     public function plural($key, $count, array $vars = [])
     {
-        if ($count == 1) {
+        if ($count == 0 || $count == 1) {
```

Comment:
Can't we make the count input var integer and have strict comparisons? All kind of things would say true here for comparison with == and 0 or 1 :)